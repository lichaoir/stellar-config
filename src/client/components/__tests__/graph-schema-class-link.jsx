const React = require('react');
const renderer = require('react-test-renderer');
const Link = require('../graph-schema-class-link.jsx');

describe('component graph-schema-class-link', () => {
  test('wraps its children', () => {
    const tree = renderer.create(<Link>content</Link>).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
