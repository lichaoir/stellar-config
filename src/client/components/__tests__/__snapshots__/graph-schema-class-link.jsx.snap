// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component graph-schema-class-link wraps its children 1`] = `
<g
  className="graph-schema-class-link"
>
  content
</g>
`;
